pkg/controller/actions/deploy/action_deploy_merge_deployment_test.go

```diff
@@ -143,3 +143,91 @@ func TestMergeDeploymentsRemove(t *testing.T) {
 		jq.Match(`.spec.template.spec.containers[0] | has("resources") | not`),
 	))
 }
+
+func TestConvertRollingUpdateFromChangeType(t *testing.T) {
+	g := NewWithT(t)
+
+	source, err := runtime.DefaultUnstructuredConverter.ToUnstructured(&appsv1.Deployment{
+		Spec: appsv1.DeploymentSpec{}, // defalt as rollingupdate
+	})
+	g.Expect(err).ShouldNot(HaveOccurred())
+	src := unstructured.Unstructured{Object: source}.Object
```

Comment:
can you just use source in this case ?

Reply:
removed some test cases.
originally was done on a case when source and target have diffrent combination.
but all in all, seems only the target with "recreate should not have rollingupdate set" matters?